lib/3scale/backend/service.rb

```diff
@@ -219,6 +219,7 @@ def storage_key(attribute)
       def delete_data
         delete_from_lists
         delete_attributes
+        ErrorStorage.delete_all(id)
```

Comment:
We call delete directly on the list in ErrorStorage because it has at most 1000 entries (defined  in MAX_NUM_ERRORS in error_storage.rb)

Reply:
It should be fine :+1: